icecream/icecream.py

```diff
@@ -330,9 +332,13 @@ def _getContext(self, callFrame, callNode):
         lineNumber = callNode.lineno
         frameInfo = inspect.getframeinfo(callFrame)
         parentFunction = frameInfo.function
-        filename = basename(frameInfo.filename)
 
-        return filename, lineNumber, parentFunction
+        if self.absPath:
+            filepath = realpath(frameInfo.filename)
+            return filepath, lineNumber, parentFunction
+        else:
+            filename = basename(frameInfo.filename)
+            return filename, lineNumber, parentFunction
```

Comment:
```suggestion
        filepath = (realpath if self.absPath else basename)(frameInfo.filename)
        return filepath, lineNumber, parentFunction
```

Reply:
That makes total sense😃
I'm updating this first thing tomorrow